modules/yieldlabBidAdapter.js

```diff
@@ -221,6 +230,35 @@ function createTargetingString (obj) {
   return str.join('&')
 }
 
+/**
+ * Creates a string out of a schain object
+ * @param {Object} schain
+ * @returns {String}
+ */
+function createSchainString (schain) {
+  let ver = schain.ver ? schain.ver : ''
+  let complete = schain.complete ? schain.complete : ''
```

Comment:
These lines can be the following. Ternary isn't needed.
```
let ver = schain.ver || ''
let complete = schain.complete || ''
```

Reply:
Thanks for the input. Fine with me!